content/momentum/3/index.md

```diff
@@ -16,4 +18,4 @@ Confidential & Proprietary.
 
 This book documents Message Systems Momentum.
 
----
+* * *
```

Comment:
This change is undoing some of the changes made here: https://github.com/SparkPost/support-docs/pull/612/files